lib/model/rules.dart

```diff
@@ -0,0 +1,18 @@
+/// A class that holds game rules
+/// 
+/// The class returns general rules using static method [generalRules()]
+/// The class returns Fou-in-a-row rules using static method [fourInARowRules()]
+/// The class returns Misere rules using static method [misereRules()]
+class GameRules {
```

Comment:
It seems it would be conciser to structure rules as a map, the same way as boardTemplates:

final gameRules = <String, String>{
  'Traditional': '...',
  'Misere': '...',
}

Reply:
I thought about it, but in this case we would access a map value by a key which is not possibly there. We would not know about it before running the program. And we would have to put "not nullable" when accessing the map value. Is it ok?

Reply:
I think it is ok. But when you create code, point me to what you mean.

Reply:
https://github.com/flutter-chat-dev/tic_tac_toe/pull/12/commits/1a686a1834fcf824f83c482333837fa3f377f412#r1182993810